src/engine/QueryExecutionTree.cpp

```diff
@@ -91,6 +91,17 @@ void QueryExecutionTree::writeResultToStream(std::ostream& out,
                                              const vector<string>& selectVars,
                                              size_t limit, size_t offset,
                                              char sep) const {
+  auto generator = generateResults(selectVars, limit, offset, sep);
+  while (generator.hasNext()) {
+    out << generator.next();
+  }
+}
```

Comment:
Is this still needed by some legacy APIs, which are not `Server.h` ?

Reply:
The only remaining usage can be found here: https://github.com/ad-freiburg/qlever/blob/8a9465c472cf72681ad1326182c0a1c7130cd799/src/SparqlEngineMain.cpp#L217-L218

Reply:
The only remaining usage can be found here: https://github.com/ad-freiburg/qlever/blob/8a9465c472cf72681ad1326182c0a1c7130cd799/src/SparqlEngineMain.cpp#L217-L218